templates/Coaster/show.html.twig

```diff
@@ -288,6 +288,18 @@
                     {% endif %}
                 </div>
             </div>
+            {% if coaster.youtubeId %}
+            <div class="panel panel-white">
+                <div class="panel-heading">
+                    <h6>
+                        {{ 'coaster.video.title'|trans }}
+                    </h6>
+                </div>
+                <div class="media-yt-container">
+                    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/{{ coaster.youtubeId }}" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
I copied the iframe provided by youtube, i checked "Enable privacy-enhanced mode" option. Apparently with this, youtube doesn't store any info about the user on the website unless the user plays the video.
I thought it would be better like that.

Reply:
good to know